--- a/setup.py
+++ b/setup.py
@@ -155,8 +155,8 @@ def append_arrow_flags(ext):
         # Arrow's manylinux{2010, 2014} binaries are built with gcc < 4.8 which predates CXX11 ABI
         # - https://uwekorn.com/2019/09/15/how-we-build-apache-arrows-manylinux-wheels.html
         # - https://arrow.apache.org/docs/python/extending.html#example
-        if "std=" not in os.environ.get("CXXFLAGS", ""):
-            ext.extra_compile_args.append("-D_GLIBCXX_USE_CXX11_ABI=0")
+        #if "std=" not in os.environ.get("CXXFLAGS", ""):
+        #    ext.extra_compile_args.append("-D_GLIBCXX_USE_CXX11_ABI=0")

     elif os.name == "nt":
         ext.extra_compile_args.append("/std:c++17")